src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/CommandLineArguments.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    public class CommandLineArguments
```

Comment:
I see, we used to pass this thing as SetOptions but now we pass a string. I stand corrected. If you're doing the work [here](https://github.com/dotnet/roslyn/pull/18258.) to open this up, why do you still need to do this?